### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by AlexaVillaume**
I updated the fluxing.rst file to add a Troubleshooting section, to address common issues. I also included fluxing in the Calibrations table of contents on the index page. I'm sure more detail can be added but this is a start.

### CODE REVIEW

File: doc/fluxing.rst [hunk 1]
```diff
@@ -48,4 +49,26 @@ respectively. The fluxed spectrum saved here is the fluxed standard,
...
+      between the knots. You can change the knot spacing by including 
+      the following in your .pypit file under the Reduce block::
+        reduce skysub bspline everyn NUM
+      where you adjust NUM. 
+    - If your observation of the standard star is taken with a setup that 
+      goes beyond the wavelength range of the version in data/standards/calspec
+    - If the wavelength solution is really bad it can manifest as problem in 
+      bspline knot. If the issue isn't the spacing or wavelength coverage check
+      the QA files to see if there is an issue in the wavelength solution. If 
+      this is the case, check the :doc:`wave_calib` page for Troubleshooting
```
[the remark this answers is not in the document]
Comment (AlexaVillaume) [reply doc/fluxing.rst]: Yeah!

**Comment by profxj** [comment]
I can't figure out what Travis is failing..

But merge when the 2 of you are happy with it.

File: doc/fluxing.rst [hunk 2]
```diff
@@ -48,4 +49,26 @@ respectively. The fluxed spectrum saved here is the fluxed standard,
...
+the bspline aglorithm. If you reach a stop in the code with a message
+that says "Problem with bspline knot" there are a couple things to check:
+
+    - There are instances where there isn't data 
+      between the knots. You can change the knot spacing by including 
+      the following in your .pypit file under the Reduce block::
+        reduce skysub bspline everyn NUM
+      where you adjust NUM. 
+    - If your observation of the standard star is taken with a setup that 
+      goes beyond the wavelength range of the version in data/standards/calspec
```
[the remark this answers is not in the document]
Comment (AlexaVillaume) [reply doc/fluxing.rst]: What should a person do if this happens?

File: doc/fluxing.rst [hunk 3]
```diff
@@ -48,4 +49,26 @@ respectively. The fluxed spectrum saved here is the fluxed standard,
...
\ No newline at end of file
+sanity check.
+
+Troubleshooting
+===============
+
+Problem with bspline knot
+-------------------------
+Things sometimes go wrong the fluxing and it commonly has to do with 
+the bspline aglorithm. If you reach a stop in the code with a message
```
[the remark this answers is not in the document]
Comment (AlexaVillaume) [reply doc/fluxing.rst]: Fixed this. Thanks for the catch.

**Comment by AlexaVillaume** [comment]
I think this PR is good to be merged for now. We can keep adding detail later.

**Comment by profxj** [comment]
merging

**Pull request merged**